src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -319,7 +318,7 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             <GenericSectionItem>
               <View accessible={true} style={styles.ticketInfoContainer}>
                 <ThemeText>
-                  {getReferenceDataName(preassignedFareProduct, language)}
+                  {getTextForLanguage(fareProductTypeConfig.name, language)}
```

Comment:
This is a bigger change that it might seem, and I don't think it's how we actually want it.

E.g. this will make 30-day periodic tickets no longer show "30-day ticket", but instead "periodic ticket, bus and tram".

Reply:
This one is not as important change, so we can revert it for now :)

Reply:
reverted